buildrunner/docker/multiplatform_image_builder.py

```diff
@@ -117,12 +117,16 @@ def __repr__(self):
 class MultiplatformImageBuilder:  # pylint: disable=too-many-instance-attributes
     """Multiple platform image builder"""
 
+    # pylint: disable=too-many-arguments
     def __init__(
             self,
+            docker_registry: Optional[str] = None,
             use_local_registry: bool = True,
             keep_images: bool = False,
             temp_dir: str = os.getcwd(),
-            disable_multi_platform: bool = False,):
+            disable_multi_platform: bool = False,
+    ):
+        self._docker_registry = docker_registry
         self._registry_info = None
```

Comment:
Maybe `self._registry_info` should be renamed to something to differentiate it more from `self._docker_registry`. `self._docker_registry` = the registry that the docker client uses to pull images from 
`self._registry_info` = is the short lived intermediate registry used for multi-platform image manifests.
So maybe rename `self._registry_info` to something like `self._mp_registry_info` or `self._mp_docker_registry`. Or something like that.